framework/core/js/src/forum/ForumApplication.ts

```diff
@@ -23,20 +23,22 @@ import isSafariMobile from './utils/isSafariMobile';
 import type Notification from './components/Notification';
 import type Post from './components/Post';
 import Discussion from '../common/models/Discussion';
```

Comment:
I know this wasn't part of your changes, but it shouldn't hurt.

```suggestion
import type Discussion from '../common/models/Discussion';
```